clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -91,6 +95,12 @@ func (p *profileMeasurement) start(config *measurement.MeasurementConfig) error
 		klog.Warning("Profile measurements will be disabled due to no MasterIps")
 		return nil
 	}
+	c := config.ClusterFramework.GetClientSets().GetClient()
```

Comment:
Any reason to put it so early in execution?
It is hard to look for "c" so maybe I am missing some use of this client. Could you use more descriptive name?

Reply:
Reason is that it's used a few lines below.

I renamed to k8sClient to make it more descriptive, thanks!

Reply:
Thx. Now I see it :)